### PR TITLE
Its good idea to unset upstream after deleting remote tracked branch

### PR DESCRIPTION
**Pull request by kkoziol**
.

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -109,7 +109,7 @@
     # Pushes the current branch to the remote "origin" (or the remote passed as the parameter) and set it to track the upstream branch
     publish = "!sh -c 'git push -u ${1:-origin} $(git branch-name)' -"
     # Deletes the remote version of the current branch from the remote "origin" (or the remote passed as the parameter)
-    unpublish = "!sh -c 'git push ${1:-origin} :$(git branch-name)' -"
+    unpublish = "!sh -c 'git push ${1:-origin} :$(git branch-name);git branch --unset-upstream $(git branch-name)' -"
```
Comment (szpak) [inline config]: Have you checked that the second command will not execute if the first fails (just a suspicion)? Maybe `&&` would be better?

In general a good idea.

**Comment by kkoziol** [comment]
Good point :/ 
Ofcourse it executes.
set -e will do.

**Comment by szpak** [comment]
Wouldn't be easier to just put `&&` instead of `;` and `set -e`?

**Comment by kkoziol** [comment]
&& does not work.

**Comment by szpak** [comment]
My heart bleeds ;).

```
unpublish2 = "!sh -c 'cat foo && echo "bar"' -"
```
results in

```
$ git unpublish2
cat: foo: No such file or directory
```
on Linux. But it's good that you found a workaround.

**Pull request merged**